src/RewardsDistributor.sol

```diff
@@ -70,7 +71,12 @@ contract RewardsDistributor is IRewardDistributor {
                 "Collateral does not match the rewards token"
             );
         }
-        payoutToken.safeTransfer(payoutTarget_, payoutAmount_);
+
+        // payoutAmount_ is always in 18 decimals precision, adjust actual payout amount to match payout token decimals
+        uint256 systemPrecision = 10 ** 18;
+        uint256 distributorPrecision = 10 ** IERC20(payoutToken).decimals();
+        uint256 adjustedAmount = (payoutAmount_ / systemPrecision) * distributorPrecision;
```

Comment:
multiply before divide is an easy way to prevent difficult to trace conditions. I bet if you write a test with a distributing ERC20 token, and distribute less than 1 token, the payout amount will be divided by 1e18 here, and it will get culled to 0

also parenthesis are unnecessary

Reply:
parenthesis are automatically added by prettier

Reply:
Updated calculations to multiply first and divide after.